README.md

```diff
@@ -1,12 +1,21 @@
-## 🚀 Important message
+## Project Overview
+Class Transcribe is a platform that students use to view lecture videos from their instructors. You can view the main repository for Class Transcribe [here](https://github.com/classtranscribe). However, Class Transcribe only comes in the form of a web app, which has many downsides, including inconveniencing students. This project focuses on creating a mobile application for Class Transcribe. This application aims to allow students to find, watch, and download lecture videos for their classes.
 
-- Gitignore .expo, ios, android
-- If changing package.json/locl/babel please communicate
-- Start working on src/Components, we will use react navigation for later (https://wix.github.io/react-native-navigation/docs/before-you-start)
-- Do Tagging
+To get involved in this project, please email angrave at illinois to learn more about how to contribute.
```

Comment:
Not sure if we want to tell people to email Angrave without his consent.

Reply:
Yeah.... that's true